src/app/components/layout/navbar/navbar.component.html

```diff
@@ -336,6 +336,29 @@
         </div>
       </div>
     </div>
+    <div class="navbar-item">
+      <div *ngIf="pingInfo.isAlive && showBellIcon && getUpdateInterval() !== '-1'">
```

Comment:
getUpdateInterval() if returns -1, then last state of showBellIcon should reset.

Otherwise it will appear as soon as value is changed to daily or weekly based on last state. While it should wait for the newest response.